utils/oomparser/oomparser.go

```diff
@@ -153,12 +153,37 @@ func (self *OomParser) analyzeLines(ioreader *bufio.Reader, outStream chan *OomI
 				line = <-lineChannel
 			}
 			in_oom_kernel_log = false
+			glog.V(1).Infof("Sending an oomInstance: %v", oomCurrentInstance)
 			outStream <- oomCurrentInstance
 		}
 	}
 	glog.Infof("exiting analyzeLines")
 }
 
+func callJournalctl() (io.ReadCloser, error) {
+	cmd := exec.Command("journalctl", "-f")
```

Comment:
Do we have access to journalctl from within the cadvisor container?
Other option might be to communicate with DBus directly eg like github.com/TheCreeper/JournalCheck